src/commands/syncback.ts

```
@@ -84,16 +92,26 @@ export async function action(commandOptions: SyncbackOptions = {}): Promise<void
 
-function buildRojoArguments(inputPath: string, syncbackOptions: SyncbackOptions): Array<string> {
+function buildRojoArguments(
+	inputPath: string,
+	syncbackOptions: SyncbackOptions,
+	config: Awaited<ReturnType<typeof loadProjectConfig>>,
+): Array<string> {
 	const args = ["syncback", "--input", inputPath, "--non-interactive"];
 
-	if (syncbackOptions.project !== undefined && syncbackOptions.project.length > 0) {
-		args.push(syncbackOptions.project);
+	const projectPath = syncbackOptions.project ?? config.rojoProjectPath;
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)